knotx-stack-manager/src/main/packaging/bin/knotx.bat

```diff
@@ -0,0 +1,105 @@
+@if "%DEBUG%" == "" @echo off 
+@rem ##########################################################################
+@rem
+@rem  vertx startup script for Windows
+@rem
+@rem ##########################################################################
+
+@rem Set local scope for the variables with windows NT shell
+if "%OS%"=="Windows_NT" setlocal
+
+@rem Add default JVM options here. You can also use JAVA_OPTS and VERTX_OPTS to pass JVM options to this script.
+set DEFAULT_JVM_OPTS=-XX:+UseBiasedLocking -XX:BiasedLockingStartupDelay=0
+
+@rem enable remote debug port, uncomment the following
+@rem set JVM_DEBUG=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
+
+set DIRNAME=%~dp0
+if "%DIRNAME%" == "" set DIRNAME=.
+set APP_BASE_NAME=%~n0
+set APP_HOME=%DIRNAME%..
+
+@rem Find java.exe
+if defined JAVA_HOME goto findJavaFromJavaHome
+
+set JAVA_EXE=java.exe
+%JAVA_EXE% -version >NUL 2>&1
+if "%ERRORLEVEL%" == "0" goto init
+
+echo.
+echo ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
+echo.
+echo Please set the JAVA_HOME variable in your environment to match the
+echo location of your Java installation.
+
+goto fail
+
+:findJavaFromJavaHome
+set JAVA_HOME=%JAVA_HOME:"=%
+set JAVA_EXE=%JAVA_HOME%/bin/java.exe
+
+if exist "%JAVA_EXE%" goto init
+
+echo.
+echo ERROR: JAVA_HOME is set to an invalid directory: %JAVA_HOME%
+echo.
+echo Please set the JAVA_HOME variable in your environment to match the
+echo location of your Java installation.
+
+goto fail
+
+:init
+@rem Add module option to commandline, if VERTX_MODS was set
+if not "%VERTX_MODS%" == "" set VERTX_MODULE_OPTS="-Dvertx.mods=%VERTX_MODS%"
+
+@rem Configure JUL using custom properties file
+if "%VERTX_JUL_CONFIG%" == "" set VERTX_JUL_CONFIG="%APP_HOME%\conf\logging.properties"
+
+@rem Get command-line arguments, handling Windowz variants
+
+if not "%OS%" == "Windows_NT" goto win9xME_args
+if "%@eval[2+2]" == "4" goto 4NT_args
+
+:win9xME_args
+@rem Slurp the command line arguments.
+set CMD_LINE_ARGS=
+set _SKIP=2
+
+:win9xME_args_slurp
+if "x%~1" == "x" goto execute
+
+set CMD_LINE_ARGS=%*
+goto execute
+
+:4NT_args
+@rem Get arguments from the 4NT Shell from JP Software
+set CMD_LINE_ARGS=%$
+
+:execute
+@rem Setup the command line
+if "%KNOTX_HOME%" == "" set KNOTX_HOME=.
+if "%KNOTX_LOGBACK_CONFIG%" == "" set KNOTX_LOGBACK_CONFIG=%KNOTX_HOME%/conf/logback.xml
+
+set CLASSPATH=%CLASSPATH%;%KNOTX_HOME%\conf;%KNOTX_HOME%\lib\*
+
+@rem Execute vertx
+"%JAVA_EXE%" %DEFAULT_JVM_OPTS% %JAVA_OPTS% %JVM_DEBUG% ^
+  -Dknotx.home=%KNOTX_HOME% ^
+  -Dlogback.configurationFile=%KNOTX_LOGBACK_CONFIG% ^
+  -Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.SLF4JLogDelegateFactory ^

```

Comment:
We need to check:
-Dhazelcast.logging.type=slf4j\
    -Dvertx.clusterManagerFactory=io.vertx.spi.cluster.impl.hazelcast.HazelcastClusterManagerFactory\
    -Dvertx.hazelcast.config=${CLUSTER_CONFIG:-${KNOTX_HOME}/conf/default-cluster.xml}\
    -Dvertx.cacheDirBase=${VERTX_CACHE_DIR:-${KNOTX_HOME}/.vertx}\
    -Dvertx.cli.usage.prefix=knotx\